metrics/classification_test.go

```diff
@@ -15,23 +15,30 @@
 package metrics
 
 import (
-	"testing"
 	"github.com/stretchr/testify/assert"
+	"testing"
 )
 
 var confusiontests = []struct {
 	in  [][]string
 	out float64
 }{
 	{
-		in: [][]string{[]string{"1", "1"}, []string{"1", "1"}, []string{"1", "1"}, []string{"1", "1"}},
+		in:  [][]string{{"1", "1", "|a"}, {"1", "1", "|a"}, {"1", "1", "|a"}, {"1", "1", "|a"}},
```

Comment:
Usually, for a change like this I would add new tests and keep existing tests to make sure existing functionality isn't broken.